<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>misure</class>
 <widget class="QDialog" name="misure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inserisci unità</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="6">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Inserisci unità</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Data di scadenza&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QLineEdit" name="id"/>
          </item>
          <item row="4" column="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Id</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Tipo unità</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="4">
           <widget class="QLineEdit" name="unita"/>
          </item>
          <item row="3" column="1" colspan="4">
           <widget class="LineEdit" name="data_scadenza"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Matricola</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QLineEdit" name="matricola"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Cerca</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Cerca</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="cerca_unita">
            <property name="text">
             <string>Cerca</string>
            </property>
            <property name="icon">
             <iconset resource="../fabaria.qrc">
              <normaloff>:/images/edit-find.png</normaloff>:/images/edit-find.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>71</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="cerca_txt"/>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Cerca per id</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cerca per tipo di unità</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lista misure</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView">
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>15</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>83</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="nuova_unita">
     <property name="text">
      <string>Nuovo</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/filenew.png</normaloff>:/images/filenew.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="save_unita">
     <property name="text">
      <string>Salva</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="del_unita">
     <property name="text">
      <string>Elimina</string>
     </property>
     <property name="icon">
      <iconset resource="../fabaria.qrc">
       <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="esci_unita">
     <property name="text">
      <string>Chiudi</string>
     </property>
     <property name="icon">
      <iconset resource="../../backup_restore_db/images.qrc">
       <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>83</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>unita</tabstop>
  <tabstop>matricola</tabstop>
  <tabstop>data_scadenza</tabstop>
  <tabstop>nuova_unita</tabstop>
  <tabstop>save_unita</tabstop>
  <tabstop>del_unita</tabstop>
  <tabstop>esci_unita</tabstop>
  <tabstop>cerca_txt</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>cerca_unita</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="../../backup_restore_db/images.qrc"/>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
